src/docs/product/cli/releases.mdx

```diff
@@ -86,11 +86,24 @@ You can still use the `--auto` flag, and the CLI will automatically use the git
 You can use the `--local` flag to enable this behavior by default.
 
 ```bash
-sentry-cli releases set-commits --local $VERSION
+sentry-cli releases set-commits "$VERSION" --local
 ```
 
 If you receive an "Unable to Fetch Commits" email, take a look at our [Help Center Article](https://help.sentry.io/hc/en-us/articles/360019866834-Why-am-I-receiving-the-email-Unable-to-Fetch-Commits-).
 
+### Dealing With Missing Commits
+
+There are scenarios in which your repositories may be missing commits previously used in the release.
+It can happen whenever you modify the commit in question, for example by ammending it, rebasing or squashing multiple commits together.
+In this case, Sentry CLI will be unable to find it, and will throw an error explaining what happened.
+
+Whenever this happens, you can pass an additional `--ignore-missing` flag, which will allow the command to fall back to the default behavior,
```

Comment:
```suggestion
When this happens, you can pass an additional `--ignore-missing` flag. This will allow the command to fall back to the default behavior,
```